### DISCUSSION TITLE
What is the correspondence between View and RenderTarget?

### DISCUSSION THREAD

**Discussion by Ankbzpx** [google/filament#4806] [Q&A] [answered]
I am confused about the correspondence between View and RenderTarget, and wonder if you could clear my confusion.

Suppose the goal is to render multiple scenes into separate renderTargets, their color/depth textures will then be composed in a Material of renderable, which will be eventually rendered into the SwapChain. Is it a good Idea to use a single View and set its Scene/RenderTarget and render it consecutively in render loop, or having separate View for each RenderTarget?

Thanks in advance!

**Comment by romainguy** [comment, marked as answer]
A View is really a holder for the scene, camera, rendering options, etc. If you are going to have different setups across render targets I would use multiple views.

**Reply by Ankbzpx** [reply]
Thanks for quick reply!
